flow-editor/src/visual-node-editor/base-node-view/style.scss

```diff
@@ -1,7 +1,7 @@
 @import "../../lib/common.scss";
 
-$node-width: 160px;
-$node-width-wide: 200px;
+$node-width: 180px;
+$node-width-wide: 240px;
```

Comment:
This changes the width of the node for all existing flows and is not an option. Node widths must stay the same.

 If it's mandatory to fix the issue, I'd expect it to be communicated explicitly. But it shouldn't be.